bentoml/archive/py_module_utils.py

```diff
@@ -12,7 +12,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-
```

Comment:
ah sorry about this line added earlier, should have run the auto format script, could you make a separate PR for this?

Reply:
yep, will do that